types/contract-proxy-kit/index.d.ts

```diff
@@ -0,0 +1,63 @@
+// Type definitions for contract-proxy-kit 1.0
+// Project: https://github.com/gnosis/contract-proxy-kit#readme
+// Definitions by: Alan Lu <https://github.com/cag>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.6
```

Comment:
Is this really the minimum version of TypeScript that is compatible with these type definitions?

Reply:
Currently, yes, because accessors were introduced in 3.6. Should I instead be making it a readonly property?

Reply:
No, that makes sense. Thanks!